biz.aQute.repository/src/aQute/bnd/repository/p2/provider/ArtifactID.java

```diff
@@ -0,0 +1,51 @@
+package aQute.bnd.repository.p2.provider;
+
+class ArtifactID {
+	private final String	id;
+	private final String	version;
```

Comment:
Why not make `version` of type `Version`? Then we avoid `toString()` calls below.